### ISSUE TITLE
Consider adding more OME metadata

### ISSUE THREAD

**Issue by nicost** [micro-manager/micro-manager#1969]
Turns out that several software packages (Omero, Arivis) have trouble with Micro-Manager data because they do not understand the well plate / position list approach taken by Micro-Manager.

We could add the option to add the Plate / well metadata to the OME metadata.  The loci library that we are using (https://javadoc.scijava.org/Bio-Formats/loci/formats/ome/OMEXMLMetadata.html) has options to set plate name/id, and well name/id/etc..  

I envision adding a "plate" option to the position list.  Once could enter a name for the plate here.  Position names can no longer be edited, allowing our code to deduce well row/column from the positonname and enter those into the OME metadata.  Optionally, the plate name can be set in the HCS Site generator.

**Comment by marktsuchida** [comment]
Agreed. I believe `MultiStagePosition` already supports arbitrary key-value metadata ("properties"). I don't remember if the HCS SG sets these, but it would make sense to do so instead of parsing position names.
